sdk/src/main/scala/com/horizen/account/utils/AccountFeePaymentsUtils.scala

```diff
@@ -0,0 +1,57 @@
+package com.horizen.account.utils
+
+import com.horizen.account.proposition.AddressProposition
+import com.horizen.evm.TrieHasher
+import com.horizen.utils.MerkleTree
+
+import java.math.BigInteger
+
+object AccountFeePaymentsUtils {
+  val DEFAULT_ACCOUNT_FEE_PAYMENTS_HASH: Array[Byte] = new Array[Byte](MerkleTree.ROOT_HASH_LENGTH)
+
+  def calculateFeePaymentsHash(feePayments: Seq[AccountBlockFeeInfo]): Array[Byte] = {
+    if(feePayments.isEmpty) {
+      // No fees for the whole epoch, so no fee payments for the Forgers.
+      DEFAULT_ACCOUNT_FEE_PAYMENTS_HASH
```

Comment:
This would all zeros, correct? The hash of an empty trie is `EMPTY_ROOT_HASH`. Does it make sense to use that instead?

Reply:
Agree, that could be a better option, I chose this because the UTXO model has the analogous constant value defined as:
`  val DEFAULT_FEE_PAYMENTS_HASH: Array[Byte] = new ArrayBytes(32)`

Reply:
Ok, so `EMPTY_ROOT_HASH` is a bit more consistent with ethereum, while all-zeroes is more consistent within the SDK? :wink:

Reply:
I set the default to EMPTY_ROOT_HASH as suggested, this is consistent with other empty tries in the block